pesacheck_meedan_bridge/py/main.py

```diff
@@ -6,13 +6,24 @@
 import settings
 from check_api import post_to_check
 from database import PesacheckDatabase, PesacheckFeed
+from lxml import etree
 from trafilatura import extract
 
 
 def html_to_text(content):
     return extract(content, include_links=True, include_images=True) or content
 
 
+def extract_summary(content):
+    tree = etree.Element("root")
+    tree.append(etree.HTML(content))
```

Comment:
If the content is missing root, why not just:
```suggestion
    tree = etree.fromstring(f"<body>{content}</body>")
```
?

Reply:
Probably because you're using XML parser which is very scrictly... switch to the HTMl one e.g.
```python
import lxml.html

tree = lxml.html.fromstring(content)

...

summary_el = figures[0].getprevious()
return summary_el.text_content()
```

Reply:
I think it's also possible to use html5parser directly but haven't tried:
```python
from lxml.html import html5parser

tree = html5parser.fromstring(content)
```